src/pagination/pagination.styles.tsx

```diff
@@ -101,98 +94,49 @@ export const NavigationButton = styled(ClickableIcon)<ButtonProps>`
         }
     }}
 
-    ${V2_MediaQuery.MaxWidth.mobileS} {
+    ${MediaQuery.MaxWidth.xxs} {
         margin-right: 0rem;
         margin-left: 0rem;
     }
 `;
 
-export const PageItem = styled(Button.Default)<StyleProps>`
+export const PageItem = styled.button<StyleProps>`
     background: ${(props) =>
-        props.$selected ? V2_Color.Primary : V2_Color.Neutral[8]};
-    border: 1px solid
-        ${(props) => (props.$selected ? V2_Color.Primary : V2_Color.Neutral[5])};
+        props.$selected ? Colour["bg-primary"] : Colour.bg};
+    border: 1px solid ${(props) => (props.$selected ? "" : Colour.border)};
```

Comment:
```suggestion
    border: 1px solid ${(props) => (props.$selected ? Colour["bg-primary"] : Colour.border)};
```

Reply:
Hi ruo ling for this i wanted to confirm if it will be Colour["bg-primary"] if selected as in figma there is no border if selected!